kitchen/ph5toms

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-#   Version: 2016.133
-EXEC='ph5tomsAPI'
+#   Version: 2017.167
+EXEC='ph5toms'
```

Comment:
what about this:
ph5tomsAPI.py --> ph5dataselectapi.py
ph5tostationxml.py --> ph5stationapi.py
ph5toexml.py --> ph5eventapi.py